.gitpod.yml

```diff
@@ -1,5 +1,24 @@
+image:
+  file: .gitpod.Dockerfile
+# Options to prebuild the image after github events and set notifications/badges
+# More here: https://www.gitpod.io/docs/prebuilds/
+github:
+  prebuilds:
+    # enable for the master/default branch (defaults to true)
+    master: true
+    # enable for pull requests coming from this repo (defaults to true)
+    pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
+    # add a check to pull requests (defaults to true)
+    addCheck: true
+    # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
+    addComment: true
+
+# List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - init: yarn
+  - before: nvm use default
+    init: yarn && alias near=./node_modules/near-shell/bin/near
```

Comment:
@willemneal I think might be more robust to use `$(yarn bin)/near` to avoid hardcoding path